core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -56,6 +57,7 @@
   private long mMountId;
   private int mInAlluxioPercentage;
   private String mUfsFingerprint = Constants.INVALID_UFS_FINGERPRINT;
+  private AccessControlList mAcl = new AccessControlList();
```

Comment:
Will this default value be ok?

Reply:
I think so? It will be set in inode and inode dir.